project.go

```diff
@@ -188,28 +191,34 @@ func (project Project) WalkTodosOfDir(dirpath string, visit func(todo Todo) erro
 		return err
 	}
 
-	scanner := bufio.NewScanner(&outb)
+	g, ctx := errgroup.WithContext(context.Background())
+	// FIXME: Arbitrary max weight for preventing "too many open files" error
```

Comment:
What's the action that is needed to be taken here I don't understand.

Reply:
So basically in an extremely large project this implementation will raise a "too many open files" error. This is related to having too many file descriptors for a process. So the action here is to find this limit and use that instead.

Also an another issue is that this implementation will use 100% of the CPU. So after reconsideration I think using a smaller worker pool is better to prevent overworking the CPU. Possibly use `runtime.NumCPU()`? What do you think?

Reply:
Why isn't it worded as `Find the max weight for preventing "too many open files" error` then?

Reply:
Thank you, I've changed the FIXME to your version in this commit https://github.com/tsoding/snitch/pull/166/commits/5169ddd507132f3d379061ace841ae512f40d5b5. It's definitely more clearer on what the issue is about. I don't know why I worded it so poorly, my bad 😅.